nasa/cumulus
# Add documentation for deploying Cumulus Distribution

**Summary:** Summary of changes

There is a corresponding change to cumulus-template-deploy open for review here: https://github.com/nasa/cumulus-template-deploy/pull/63

This is user-facing documentation for how to configure and deploy the new Cumulus Distribution API. There is also a page on the wiki that should be reviewed as part of this PR:

https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus+Distribution+Options

Testing instructions:

1. Follow the directions starting from docs/deployment/README.md#cumulus-instance-sizing to deploy using the new distribution API and Cognito. Use the `cumulus-template-deploy` repo with these changes: https://github.com/nasa/cumulus-template-deploy/pull/63
2. Test ingesting and publishing a granule. The file from CMR should contain links to download the granule files from S3 via the new distribution API. Ping me for help testing this.
3. Re-deploy with the TEA variables set and test that the distribution app is switched to TEA by running the same workflow and looking for the TEA URL in the CMR file.

I'm very open to adding to these docs. I'm working under the assumption that we're mainly concerned with documenting deployment, not so much Cognito configuration (as we don't control that). It feels weird to leave that piece out entirely though so this might be an area that needs input.

## PR Checklist
